src/controls/widgets/group/WidgetGroup.tsx

```diff
@@ -90,7 +90,7 @@ export class Widget_group<T extends SchemaDict> implements IWidget<Widget_group_
             type: 'group',
             id: this.id,
             active: true,
-            collapsed: this.config.startCollapsed ?? false,
+            collapsed: this.config.startCollapsed,
```

Comment:
This should not be collapsed by default, allow it to remain `undefined` so that the widget can control the unset default collapsed state (based on if it is active)

Reply:
I think active is a legacy property here, and should be unused, but I see your point. You want to fix optionals being unfolded by default ? There is very probably a default to change, not 100% sure this is here